### ISSUE TITLE
Release, please

### ISSUE THREAD

**Issue by thet** [collective/collective.honeypot#5]
@mauritsvanrees @fredvd a new release of this package would be very awesome!
If you want to give me maintainer rights on pypi, my username is: thet

**Comment by mauritsvanrees** [comment]
Hi @thet. Thanks for you work on this! I haven't taken the time to properly test your improvements, and see if more fixes may be needed, but it is certainly an improvement that this package now works on Plone 5.2. Go ahead and make a 2.0.0 release, I made you Owner on PyPI.

**Comment by thet** [comment]
hi @mauritsvanrees 
i can't fond myself being an owner at: https://pypi.org/project/collective.honeypot/

**Comment by mauritsvanrees** [comment]
In the security history I see I sent an invite. I think the you cannot add an Owner directly, but have to do this via an invite, which is only valid for a week or so.
Let me try again. Done.

**Comment by thet** [comment]
oh-my, I missed that. sorry for the noise and thanks!

**Comment by stevepiercy** [comment]
:hugs: for the release. I think this issue can now be closed, yes?
